addon/macros/feature-flag.js

```diff
@@ -0,0 +1,21 @@
+import { camelize } from '@ember/string';
+import { computed, get, set } from '@ember/object';
+import { getOwner } from '@ember/application';
+
+const featuresServiceProperty = `_features_${+(new Date())}`
+
+export default function featureFlagMacro(flag) {
```

Comment:
What if this required passing the name of the features service injection, defaulting it to 'features':
`export default function featureFlagMacro(flag, featureServiceName = 'features') {`

Reply:
If it must. I'd prefer not to have to declare the computed property _and_ the service injection. It's a lot of boilerplate for a single property that depends on a feature flag.

Reply:
Yeah, but we've done a lot of work to get away from "magic" injection. I'd like to leave it to the user.